frontend/src/services/session.test.ts

```diff
@@ -39,33 +37,4 @@ describe("startNewSession", () => {
     expect(setupSpy).toHaveBeenCalledTimes(1);
     expect(sendSpy).toHaveBeenCalledWith(JSON.stringify(event));
   });
-
-  it("should start with the custom llm if set", () => {
-    const settings: Settings = {
-      LLM_MODEL: "llm_value",
-      CUSTOM_LLM_MODEL: "custom_llm_value",
-      USING_CUSTOM_MODEL: true,
-      AGENT: "agent_value",
-      LANGUAGE: "language_value",
-      LLM_API_KEY: "sk-...",
-      CONFIRMATION_MODE: true,
-      SECURITY_ANALYZER: "analyzer",
-    };
-
-    const event = {
-      action: ActionType.INIT,
-      args: settings,
-    };
-
-    saveSettings(settings);
-    Session.startNewSession();
-
-    expect(setupSpy).toHaveBeenCalledTimes(1);
-    expect(sendSpy).toHaveBeenCalledWith(
-      JSON.stringify({
-        ...event,
-        args: { ...settings, LLM_MODEL: "custom_llm_value" },
-      }),
-    );
-  });
```

Comment:
Should we create a new test for the new behavior?

Reply:
Sorry got excited and hit merge--this would have been a good idea! It's pretty well covered by existing cases but it'd be good to have an explicit one. I'll put a new test in a follow-up PR